package/base-files/files/bin/config_generate

```diff
@@ -318,6 +318,7 @@ generate_led() {
 			uci -q batch <<-EOF
 				set system.$cfg.trigger='netdev'
 				set system.$cfg.mode='$mode'
+				set system.$cfg.interval='50'
```

Comment:
Please drop this line. 50 is the default value anyway and I don't see the point in adding a fixed value here.